src/Our.Umbraco.AuthU/Data/Migrations/UmbracoDbOAuthClientStore/Add/CreateDemoClient.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Our.Umbraco.AuthU.Models;
+using Umbraco.Core.Migrations;
+
+namespace Our.Umbraco.AuthU.Data.Migrations.UmbracoDbOAuthClientStore
+{
+    /// <summary>
+    /// Simple migration to add a demo client to the OAuthClient Table
+    /// </summary>
+    internal class CreateDemoClient : MigrationBase
+	{
+		public CreateDemoClient(IMigrationContext context) : base(context)
+		{ }
+
+        public override void Migrate()
+        {
+            var existing = Database.SingleOrDefault<OAuthClient>($"SELECT * FROM [OAuthClient] WHERE [ClientId] = @0", "DemoClient");
```

Comment:
I'd probably change this query to `SELECT 1 FROM` as we don't actually need any data, we are just checking if anything exists. Fetching all the data from the DB is a waste here so lets make this as slim as we can.